src/core/state.py

```
@@ -5,14 +5,11 @@
 from pydantic import BaseModel, Field
 import croniter
 from pathlib import Path
-from ._types import SubmitSequenceData, SequenceData, JobExecutuionData, SequenceResult, SequenceStatus, JobState, JobState
+from ._types import SubmitSequenceData, JobExecutuionData, SequenceResult, SequenceStatus, JobState
```

Ruff (0.12.2)

`._types.SubmitSequenceData` imported but unused

Remove unused import

(F401)